cdap-common/src/main/java/co/cask/cdap/common/conf/KafkaConstants.java

```diff
@@ -25,15 +25,35 @@
    * Keys for configuration parameters.
    */
   public static final class ConfigKeys {
+    public static final String NUM_PARTITIONS_CONFIG = "kafka.server.num.partitions";
+    public static final String LOG_DIR_CONFIG = "kafka.server.log.dir";
+    public static final String REPLICATION_FACTOR = "kafka.server.default.replication.factor";
+    public static final String FLUSH_INTERVAL = "kafka.server.log.flush.interval.messages";
```

Comment:
We don't need all these constants (this and below). If we want to provide default values, just put them in `cdap-default.xml`